<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="replot_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>rePlot</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>x max</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>y min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLineEdit" name="y_min_line_edit">
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>x min</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="line_func"/>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>F(x)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLineEdit" name="x_max_line_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>y max</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="x_min_line_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>60</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QLineEdit" name="y_max_line_edit">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Кол-во точек</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QLineEdit" name="line_points_num"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
